src/dataBase.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', 'test123', {
```

Comment:
Avoid hardcoding credentials and database details. Use environment variables or a configuration file.